src/SampSharp.GameMode/Definitions/MapIcons.cs

```diff
@@ -0,0 +1,395 @@
+// SampSharp
+// Copyright 2017 Tim Potze
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace SampSharp.GameMode.Definitions
+{
+    
+    /*
```

Comment:
Could you remove the comment with the JS how you scraped the values? Not really necessary here and if you instead paste it in the PR as a comment we can always come back to it.